code/game/area/areas.dm

```diff
@@ -120,9 +120,13 @@
 
 	/// List of all air vents in the area
 	var/list/obj/machinery/atmospherics/components/unary/vent_pump/air_vents = list()
+	/// List of air vents in the area, that shouldn't be touched by air alarm modes
+	var/list/obj/machinery/atmospherics/components/unary/vent_pump/excluded_vents = list()
 
 	/// List of all air scrubbers in the area
 	var/list/obj/machinery/atmospherics/components/unary/vent_scrubber/air_scrubbers = list()
+	/// List of air scrubbers in the area, that shouldn't be touched by air alarm modes
+	var/list/obj/machinery/atmospherics/components/unary/vent_scrubber/excluded_scrubbers = list()
```

Comment:
why have this be a list on the area? couldn't we store this on the vent/scrubber itself? is there a side effect of this that you prefer?

Reply:
made it easier to just subtract one list from the other right in 'for' loop rather than piling 'if var then early return' to every single mode twice